content/post/Android_Root101.en.md

```diff
@@ -85,13 +92,12 @@ _After completing the process make sure to clear the Dalvik Cache and Cache memo
 
 _Reboot the Device._
 
-**_Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares._**
+{{<yellowborder>}}Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares.{{</yellowborder>}}
 
-______
 ______
 
-*If during the process you encoutered any issues do look up your issues on XDA or else Google your issues, you'll surely find a solution.*
+*If during the process you encoutered any issues, do look up your issues on XDA or else Google your issues, you'll surely find a solution.*
```

Comment:
Woah, did you just tell me "**If you find any issues in the process I taught you,  go fix them yourself**" ?
Because that's exactly what it means.

It can easily be fixed. All you need is a very small change.